### PR TITLE
[chore]: Testing cleanup -- ember-native-dom-helpers

### PR DESCRIPTION
**Pull request by alexander-alvarez**
As promised. Moving to use `ember-native-dom-helpers` instead of jQuery selectors.
We should investigate using them in the addon code to make it jQuery-less

### CODE REVIEW

File: tests/integration/components/light-table-test.js [hunk 1]
```diff
@@ -174,24 +170,23 @@ test('passed in components can have computed properties', function(assert) {
...
   this.set('current', users[0]);
-
-  assert.ok(this.$('.custom-row:eq(0)').hasClass('is-active'), 'first custom row is active');
+  let [firstRow] = findAll('.custom-row');
+  assert.ok(hasClass(firstRow, 'is-active'), 'first custom row is active');
 
   this.set('current', users[2]);
-
-  assert.ok(this.$('.custom-row:eq(2)').hasClass('is-active'), 'third custom row is active');
+  let [,, thirdRow] = findAll('.custom-row');
```
Comment (buschtoens) [inline tests/integration/components/light-table-test.js]: Personally, I would have probably gone for 

```js
let thirdRow = findAll('.custom-row')[3];
```

or even

```js
let thirdRow = find('.custom-row:nth-child(3)');
```

Line 177 could then also be:

```js
let firstRow = find('.custom-row:nth-child(1)');
```

What do you think?

File: tests/integration/components/lt-body-test.js [hunk 2]
```diff
@@ -33,73 +34,66 @@ test('row selection - enable or disable', function(assert) {
...
+  assert.ok(hasClass(row, 'is-selected'));
 });
 
 test('row selection - ctrl-click to modify selection', function(assert) {
   this.set('table', new Table(Columns, createUsers(5)));
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=true multiSelect=true}}`);
+  let rows = findAll('tr');
+  let [firstRow, , , middleRow] = rows;
+  let lastRow = rows[rows.length - 1];
```
Comment (buschtoens) [inline tests/integration/components/lt-body-test.js]:
```js
let firstRow = find('tr:first-child');
let middleRow = find('tr:nth-child(4)');
let lastRow = find('tr:last-child');
```

I think that's a bit more expressive. You can tell at a glance what index `middleRow` has.

File: tests/integration/components/lt-body-test.js [hunk 3]
```diff
@@ -33,73 +34,66 @@ test('row selection - enable or disable', function(assert) {
...
   this.set('table', new Table(Columns, createUsers(5)));
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=true multiSelect=true multiSelectRequiresKeyboard=false}}`);
 
-  let firstRow = this.$('tr:first');
-  let middleRow = this.$('tr:nth-of-type(3)');
-  let lastRow = this.$('tr:last');
+  let rows = findAll('tr');
+  let [firstRow, , , middleRow] = rows;
+  let lastRow = rows[rows.length - 1];
```
Comment (buschtoens) [inline tests/integration/components/lt-body-test.js]:
```js
let firstRow = find('tr:first-child');
let middleRow = find('tr:nth-child(4)');
let lastRow = find('tr:last-child');
```

Same.

File: tests/integration/components/lt-body-test.js [hunk 4]
```diff
@@ -112,28 +106,29 @@ test('row expansion', function(assert) {
     {{/lt-body}}
   `);
 
-  let row = this.$('tr:first');
+  let [row] = findAll('tr');
```
Comment (buschtoens) [inline tests/integration/components/lt-body-test.js]: Equivalent to

```js
let row = find('tr');
```

File: tests/integration/components/lt-body-test.js [hunk 5]
```diff
@@ -165,30 +159,30 @@ test('row actions', function(assert) {
   this.on('onRowDoubleClick', (row) => assert.ok(row));
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions onRowClick=(action 'onRowClick') onRowDoubleClick=(action 'onRowDoubleClick')}}`);
 
-  let row = this.$('tr:first');
-  row.click();
-  row.dblclick();
+  let [row] = findAll('tr');
```
Comment (buschtoens) [inline tests/integration/components/lt-body-test.js]: Equivalent to

```js
let row = find('tr');
```

File: tests/integration/components/lt-head-test.js [hunk 6]
```diff
@@ -12,32 +13,33 @@ test('render columns', function(assert) {
...
+    assert.notOk(column.sortable);
     assert.equal(column.label, 'Avatar');
   });
 
   this.render(hbs`{{lt-head table=table renderInPlace=true onColumnClick=(action 'onColumnClick')}}`);
 
-  assert.equal(this.$('tr > th').length, 6);
-  this.$('tr > th:first').click();
+  assert.equal(findAll('tr > th').length, 6);
+  let [nonSortableHeader] = findAll('tr > th');
```
Comment (buschtoens) [inline tests/integration/components/lt-head-test.js]: Equivalent to

```js
let row = find('tr > th');
```

File: tests/integration/components/lt-body-test.js [hunk 7]
```diff
@@ -33,73 +34,66 @@ test('row selection - enable or disable', function(assert) {
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=canSelect}}`);
 
-  let row = this.$('tr:first');
+  let [row] = findAll('tr');
```
Comment (buschtoens) [inline tests/integration/components/lt-body-test.js]: Equivalent to

```js
let row = find('tr');
```

File: tests/integration/components/light-table-test.js (hunk 1 above)
Comment (alexander-alvarez) [inline tests/integration/components/light-table-test.js]: I started with the array indexing, but then `ember-suave` complained about it and I wasn't in the mood to investigate right now. will swap in `nth-child` selectors

**Pull request merged**